operations/security_and_authentication/Audit_Logs.md

```diff
@@ -0,0 +1,112 @@
+# Audit Logs
+
+## Enable or disable audit logging for host and Kubernetes APIs
+
+Audit logs are used to monitor the system and search for suspicious behavior.
+Host and Kubernetes API audit logging can be enabled to produce extra audit logs for analysis.
+Enabling audit logging is optional. If enabled it generates some load and data on the non-compute nodes \(NCNs\).
+By default, host and Kubernetes API audit logging are not enabled.
+
+### Procedure
+
+The method for updating the audit log settings varies depending on the state of the system.
+
+1. Select one of the following scenarios for the state of system accordingly:
+
+   * Install (`ncn#`)
```

Comment:
release/1.2 uses the old command prompt conventions, just like release/1.0. So these should be changed accordingly.